sockeye/arguments.py

```diff
@@ -430,11 +430,21 @@ def add_training_args(params):
                               help='Minimum number of epochs (passes through the training data) '
                                    'before fitting is stopped. Default: %(default)s.')
 
-    train_params.add_argument('--dropout',
+    train_params.add_argument('--embed-dropout',
+                              type=multiple_values(2, data_type=float),
+                              default=(.0, .0),
+                              help='Dropout probability for source & target embeddings. Use <val>:<val> to specify '
+                                   'separate values. Default: %(default)s.')
+    train_params.add_argument('--rnn-dropout',
+                              type=multiple_values(2, data_type=float),
+                              default=(.0, .0),
+                              help='RNN variational dropout probability for encoder & decoder RNNs.'
+                                   'Use <val>:<val> to specify separate values. Default: %(default)s.')
+
+    train_params.add_argument('--rnn-decoder-hidden-dropout',
                               type=float,
-                              default=0.,
-                              help='Dropout probability for source embedding and source and target RNNs. '
-                                   'Default: %(default)s.')
+                              default=.0,
+                              help='Dropout probability for hidden state in RNN decoder. Default: %(default)s.')
```

Comment:
I will change to the updated description.
I don't have a better idea for the argument name but think its good enough. 'rnn' stands for the attentional rnn model anyway now in contrast to 'transformer' so that should be fine.